spec/system/admin/orders_spec.rb

```diff
@@ -223,12 +223,31 @@
             login_as_admin_and_visit spree.admin_orders_path
           end
           it "orders by completion date" do
-            find("a", :text => 'COMPLETED AT').click # sets ascending ordering
+            find("a", text: 'COMPLETED AT').click # sets ascending ordering
             expect(page).to have_content "Loading"
-            expect(page).to have_content (/#{order5.number}.*#{order4.number}.*#{order3.number}.*#{order2.number}/m)
-            find("a", :text => 'COMPLETED AT').click # sets descending ordering
+            expect(page).to have_content(/#{order5.number}.*#{order4.number}.*#{order3.number}.*#{order2.number}/m)
+            find("a", text: 'COMPLETED AT').click # sets descending ordering
             expect(page).to have_content "Loading"
```

Comment:
This is only shown for a short amount of time, right? This can lead to a race condition where "Loading" already disappeared when this expectation is executed and we got a flaky spec.

Reply:
Oh, now I see that you removed it again. That was a detour.

Reply:
Yes, it was indeed a detour - I noticed later on that specs were failing because of this, so I've removed it :+1: